ViAn/GUI/projectwidget.cpp

```diff
@@ -18,26 +22,40 @@ void ProjectWidget::new_project() {
     }
 }
 
+/**
+ * @brief ProjectWidget::add_project
+ * Slot function called by the create project dialog.
+ * Creates a new project and generates the default tree structure
+ * @param project_name
+ * @param project_path
+ */
 void ProjectWidget::add_project(QString project_name, QString project_path) {
```

Comment:
perhaps this should be named "new_project", if we load an existing project we should probably not create default tree for example